python/python/Ice/__init__.py

```diff
@@ -113,3 +113,108 @@
 # Native PropertiesAdmin admin facet.
 #
 NativePropertiesAdmin = IcePy.NativePropertiesAdmin
+
+__all__ = [
```

Comment:
__all__ defines the the public API, used by `import *` and by sphinx to figure out what types are documented with automodule.

Lot of types where missing in the previous setup.

Reply:
Would be nice to add a comment in the code.